Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest
+
+LABEL ORG="Armedia LLC" \
+      APP="Alfresco" \
+      VERSION="1.0" \
+      IMAGE_SOURCE=https://github.com/ArkCase/ark_mongodb \
+      MAINTAINER="Armedia LLC"
+
+ENV MONGO_USERID=2000 \
+    MONGO_GROUPID=2020 \
+    MONGO_GROUPNAME=mongod \
+    MONGO_USER=mongod 
+
+ARG MONGO_VERSION=5.0.6
+#Install dependencies
+
+RUN yum update -y && yum install -y libcurl openssl xz-libs &&\
+    groupadd -g ${MONGO_GROUPID} ${MONGO_GROUPNAME} && \
+    useradd -u ${MONGO_USERID} -g ${MONGO_GROUPNAME} ${MONGO_USER}
+ENV MONGO_URL="https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-rhel70-${MONGO_VERSION}.tgz"
+ENV MONGOSH_URL="https://downloads.mongodb.com/compass/mongosh-1.2.2-linux-x64.tgz"
+
+
+WORKDIR /opt
+ADD ${MONGO_URL} /opt/
+ADD ${MONGOSH_URL} /opt/
+RUN set -ex;\
+    tar -xzvf mongodb-linux-x86_64-rhel70-${MONGO_VERSION}.tgz; tar -xzvf mongosh-1.2.2-linux-x64.tgz ;\
+    rm mongodb-linux-x86_64-rhel70-${MONGO_VERSION}.tgz mongosh-1.2.2-linux-x64.tgz;\
+    ln -s $(pwd)/mongodb-linux-x86_64-rhel70-${MONGO_VERSION}/bin/* /usr/local/bin/ ;\
+    ln -s $(pwd)/mongosh-1.2.2-linux-x64/bin/* /usr/local/bin/ ;\
+    mkdir -p /var/lib/mongo; mkdir -p /var/log/mongodb; \
+    chown -R  ${MONGO_USER}:${MONGO_USER} /var/lib/mongo; chown -R  ${MONGO_USER}:${MONGO_USER} /var/log/mongodb;
+EXPOSE 27017
+USER ${MONGO_USER}
+
+CMD [ "mongod","--dbpath","/var/lib/mongo","--logpath","/var/log/mongodb/mongod.log"]
```

Comment:
I believe Docker images should log to `/dev/stdout` or otherwise configure logging to output only to the standard output.  I see some or all our other containers don't configure logging at all.  This will be a minimum-viable-product issue; we need to figure out how containers should log, then make sure they all log in that way.

Reply:
@david-oc-miller if we omit --logpath in CMD all logs would be on /dev/stdout

Reply:
@tSudharshan , if you omit `--logpath` does it still go to `/var/log/mongodb` as the default path, or does it definitely go to standard output only?

Reply:
@david-oc-miller  it goes to standard  out